tests/integration/security/reachability_test.go

```diff
@@ -60,7 +61,7 @@ func TestReachability(t *testing.T) {
 			integIstioVersion := cMinIstioVersion
 			var migrationApp echo.Instances
 			// if dual stack is enabled, a dual stack echo config should be added
-			if !t.Settings().EnableDualStack {
+			if len(t.Settings().IPFamilies) <= 1 {
```

Comment:
You can drop this `if/else` and just use `IPFamilies: strings.Join(t.Settings().IPFamilies, ","),` for both cases, I believe, the two definitions are exactly the same otherwise.